src/components/PeopleFilters.tsx

```diff
@@ -27,66 +67,48 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuriesList.map(century => (
+              <SearchLink
+                key={century}
+                data-cy="century"
+                className={cn('button', 'mr-1', {
+                  'is-info': searchParams.getAll(SearchField.Centuries)
+                    .includes(century),
+                })}
+                params={{
+                  centuries:
+                    !searchParams.getAll(SearchField.Centuries)
+                      .includes(century)
+                      ? [...searchParams.getAll(SearchField.Centuries), century]
+                      : searchParams.getAll(SearchField.Centuries)
+                        .filter(c => c !== century),
```

Comment:
- `searchParams.getAll(SearchField.Centuries).includes(century)` you do it twice, create a variable
- avoid one-letter names for variables/parameters